[RA2 Ch6]: Clarifications to container image handling (#1712)
cntt-n/CNTT: doc/ref_arch/kubernetes/chapters/chapter05.md

- Image registries are monitored to automatically block or replace images with known vulnerabilities, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
- Adopt a layered packaging model which supports separation of concerns during image build.

> The following functionalities are recommended for securing kubernetes platform:

Reviewer: ```suggestion
The following functionalities are recommended for securing Kubernetes platforms:
```

> - Image Certification (Scan for voulnerabilities) and Signing

Reviewer: ```suggestion
- Image Certification (Scan for vulnerabilities) and Signing
```

- Role-base Access Control

> - Secret Managment

Reviewer 2: ```suggestion
- Secret Management
```

##  5.13  Trusted Registry
Ensure that the container registry only accepts container images from trusted sources that have tested and validated the images. Where images are provided by third parties, define and follow a formal process to validate compliance with security requirements. Also ensure that access control is applied to registries requiring unique credentials, to limit who can control the build or add images.

> - Network access to the registry must run over TLS or VPN connections

Reviewer 3: Can the 'must' be changed to guidance / principle wording please?

Author: Humm, not sure. Can we trust a registry if the connection is optionally secure to it? What others think?

Reviewer: I would agree with @CsatariGergely

Reviewer 4: I would agree "in principle" with @CsatariGergely and @xmulligan. My suggestion use the following statement:
"Network access to the registry must run over a secured connections, such as TLS, SSl, or VPN."
Just to  not limit to a particular protocol.
@CsatariGergely, please address this discussion and we will accept this PR.

Reviewer 3: Would the following work?  I don't disagree that it's needed, it's just the language I'm wary of as this isn't part of the specification, it's guidance.
```suggestion
- It is strongly recommended that network access to the registry is secured using TLS or VPN connections to ensure trust.
```
